test/e2e/e2e_test.go

```diff
@@ -41,7 +41,12 @@ type Clients struct {
 }
 
 func TestMasterMachineCreated(t *testing.T) {
-	clients, err := createTestClients()
+	kube_config string = "KUBE_CONFIG"
```

Comment:
```suggestion
```

Reply:
I have removed this line

Reply:
I still see it. It hasn't reflected here in this PR.

Reply:
I did git status and it shows nothing to commit, working tree clean.

Reply:
did you push the code?

Reply:
Have you pushed the changes?

Reply:
Hi @tariq1890 . I have put up a link of a screenshot for the same issue. PTAL. 
[https://drive.google.com/file/d/1FOTg20rx3mhVkZXVF7jvjRIOhPlTXYcU/view?usp=sharing](url)

Reply:
[https://drive.google.com/file/d/1FOTg20rx3mhVkZXVF7jvjRIOhPlTXYcU/view?usp=sharing](url)

Reply:
@Avni-Sharma -- try this:

```bash
git remote update # updates your remotes
git rebase upstream/master # rebase your branch on top of upstream

# Make any code changes
git commit <files changed>
git push -f <your_remote> <your_branch> # force-push your local changes
```

Reply:
Yeah. I will try it today